src/library.js

```diff
@@ -2090,6 +2094,13 @@ addToLibrary({
     console.error(UTF8ToString(str));
   },
 
+  emscripten_console_trace: (str) => {
+#if ASSERTIONS
+    assert(typeof str == 'number');
```

Comment:
wait... why is it called 'str' if it's supposed to be a number? it seems to work fine with regular strings in node?

Reply:
These are C APIs, so this is a C string, AKA a pointer, AKA a number.

Reply:
Ah of course.